pkg/test/harness.go

```diff
@@ -310,6 +310,16 @@ func (h *Harness) Stop() {
 		h.managerStopCh = nil
 	}
 
+	if h.kind != nil {
+		logDir := filepath.Join(h.TestSuite.ArtifactsDir, fmt.Sprintf("kind-logs-%d", time.Now().Unix()))
```

Comment:
so the artifactsdir is right now used only for kind logs and no other logs? Should we document that somewhere?

Reply:
Yeah, agreed, I'll add a doc snippet somewhere that specifies what artifacts get collected.